packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -169,6 +187,11 @@ class HttpRouteSpec extends RouteSpec {
    */
   public readonly match?: HttpRouteMatch;
 
+  /**
+   * The criteria for determining a timeout configuration
+   */
```

Comment:
Needs `@default` annotation

Reply:
hm so this needs a annotation but `public readonly match?: HttpRouteMatch;` doesnt?

Reply:
Ah, sorry I thought this was in a different class. you're fine here!